### DISCUSSION TITLE
Pack of multiple transmitters and 1 receiver

### DISCUSSION THREAD

**Discussion by Yasshion** [jgromes/RadioLib#875] [Q&A]
Hello i was wondering about SX1262 settings. I have multiple senders and a reciever. Its no problem with communicating because they are sending at different time but i wanted to know if i add another pack of multiple and reciever for relative close area. How can i assure recievers get their respective transmitters' message? Maybe adding a unique id for reciever that will be sent from its' respective transmitters' so i can filter out to now mix it up from other pack. There will be couple pack of transmitters and recievers so it is important to recievers' get correct messages from their respective transmitters. 
For example: there will be 2 receivers not far from each other 50 meter far aways but  "transmitters node A" will send the data "Receiver A", "transmitters B's" message will sent it to "Receiver B". I was looking for the sync word or something that defines the lora address. I want to give them a unique id random generated id from "abcdefghijklmnoprsxyz0123456789" i would like to use this as address is it possible?.

**Comment by jgromes** [comment]
You have a couple options:

1. Use a different frequency - if the two networks have a sufficiently different frequency, they won't interfere with each other, Keep in mind you local regulations may limit which frequencies you can use.
2. Use a different spreading factor - if the networks use different spreading factors, they shouldn't interfere. The downside here is that spreading factors will lead to different bit rates and sensitivity/range.
3. Use ID filtering, as you suggested - by itself, LoRa has no concept of a "node address", and its sync word is not 100% reliable. The receivers would receive everything and only accept messages from some subset of transmitters. The advantage here is that to reconfigure the network, you only have to change the receivers

Any of these should be able to do what you need, though each has its own advantages and drawbacks.

**Reply by Yasshion** [reply]
Thank you for answering. Transmitter sends device id first then the data for example 
Device1helloworld
#Define Device_ID Device1   used 
if (receivedData.startsWith(Device_ID)
{
Serial.println(ReceivedData);
}else {
          Serial.println(F("Device ID mismatch. Ignoring data..."));
        }
        it worked with this way
